sdk/nodejs/runtime/rpc.ts

```diff
@@ -206,7 +206,7 @@ export function resolveProperties(
     for (const k of Object.keys(resolvers)) {
         if (!allProps.hasOwnProperty(k)) {
             const resolve = resolvers[k];
-            resolve(undefined, !isDryRun(), false);
+            resolve(isDryRun() ? unknown : undefined, !isDryRun(), false);
```

Comment:
is this technically necessary?  since we're resolving the output to isKnown=false, wouldn't the val actually be massaged to be hte "unknown" value as well?

Reply:
Nope, this isn't technically necessary. I can change this back.